configurePipelineFiles.sh

```diff
@@ -0,0 +1,226 @@
+  printf "\\e[38;5;27m"; base64_print CiAgICBfX18gXyAgICAgICAgICAgIF8gXyAgICAgICAgICAgICAgX19fX18gICAgICAgICAgICBfIAogICAvIF8gKF8pXyBfXyAgIF9fX3wgKF8pXyBfXyAgIF9fXyAgL19fICAgXF9fXyAgIF9fXyB8IHwKICAvIC9fKS8gfCAnXyBcIC8gXyBcIHwgfCAnXyBcIC8gXyBcICAgLyAvXC8gXyBcIC8gXyBcfCB8CiAvIF9fXy98IHwgfF8pIHwgIF9fLyB8IHwgfCB8IHwgIF9fLyAgLyAvIHwgKF8pIHwgKF8pIHwgfAogXC8gICAgfF98IC5fXy8gXF9fX3xffF98X3wgfF98XF9fX3wgIFwvICAgXF9fXy8gXF9fXy98X3wKICAgICAgICAgfF98ICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCg==
+  printf "\\e[0m"; printf '\n'
+  echo -e "${BOLD}${WELCOME_MSG}${RESET}\\n"
+}
+
+import_manager_prompts() {
+  TEST_IMPORT_PROPERTIES=$CURRENT_DIR/devops/adm/import-manager.test.properties
+  STAG_IMPORT_PROPERTIES=$CURRENT_DIR/devops/adm/import-manager.stag.properties
+  PROD_IMPORT_PROPERTIES=$CURRENT_DIR/devops/adm/import-manager.prod.properties
+
+  # Read in the URL's of all the environments
+  read -p "$(echo -e $WHITE)Enter URL of your test environment (ending with /suite):$(echo -e $RESET) " TEST_URL
+  echo -e "\\n${BLACK_ON_GREY}Modified $TEST_IMPORT_PROPERTIES with the given url${RESET}\\n"
+  read -p "$(echo -e $WHITE)Enter URL of your staging environment (ending with /suite):$(echo -e $RESET) " STAG_URL
+  echo -e "\\n${BLACK_ON_GREY}Modified $STAG_IMPORT_PROPERTIES with the given url${RESET}\\n"
+  read -p "$(echo -e $WHITE)Enter URL of your production environment (ending with /suite):$(echo -e $RESET) " PROD_URL
+  echo -e "\\n${BLACK_ON_GREY}Modified $PROD_IMPORT_PROPERTIES with the given url${RESET}\\n"
+
+  # Configure import manager properties files with the given site URL's (only if non-empty)
+  if [[ $TEST_URL ]]; then
+    sed-populate-field "url" "$TEST_URL" $TEST_IMPORT_PROPERTIES
+  fi
+  if [[ $STAG_URL ]]; then
+    sed-populate-field "url" "$STAG_URL" $STAG_IMPORT_PROPERTIES
+  fi
+  if [[ $PROD_URL ]]; then
+    sed-populate-field "url" "$PROD_URL" $PROD_IMPORT_PROPERTIES
+  fi
+}
+
+version_manager_prompts() {
+  VERSION_MANAGER_PROPERTIES=$CURRENT_DIR/devops/adm/version-manager.properties
+
+  echo -e "\\n${BLACK_ON_GREY}Modifying $VERSION_MANAGER_PROPERTIES with the following prompts${RESET}\\n"
+  read -p "$(echo -e $WHITE)Enter your repo URL (repo where you want to version your applications):$(echo -e $RESET) " REPO_URL
+  read -p "$(echo -e $WHITE)Enter a folder name where you want this repo to be cloned (ex: staging):$(echo -e $RESET) " LOCAL_REPO
+  read -p "$(echo -e $WHITE)Enter a name of the branch you want to clone from your repo (Default is master):$(echo -e $RESET) " BRANCH_NAME
```

Comment:
I wanted to leave this in for when we test internally too. Like I have a private gitlab repo where I host all the different changes of applications/configuration files/Jenkinsfile on different branches and I want the pipeline to use that branch. 

Leaving this out would mean there's one manual change to make when setting up all the files for testing. Unless we envision that if you are running this script, you only mean to use the master branch.

Reply:
I think its fair to have one that we use internally and we could distribute that to everyone and add to .gitignore, but since we have a clear recommended strategy that utilizes only master, I think we want to remove this. We also don't mention branching in the doc, so it may be confusing